stories/topics/proc-aws-running-upgrade.adoc

```diff
@@ -1,6 +1,12 @@
 [id="proc-aws-running-upgrade"]
 
-= Running the ansible-on-clouds-ops 2.4 container to upgrade your {PlatformNameShort} deployment
+= Running the upgrade playbook
+
+[NOTE]
+=====
+{PlatformNameShort} upgrade to 2.4.20230630 will update the listeners on its internal load balancers. If a resource dependancy was added after installation, this will need to be temporarily removed to allow the upgrade to succeed. See xref:tech-note-aws-upgrade[technical note] for more detail.
```

Comment:
{PlatformNameShort} upgrade to 2.4.20230630 updates the listener...

after installation, this must be temporarily removed for the upgrade to succeed.